### PR TITLE
[chore] Add issue templates

### PR DESCRIPTION
**Pull request by arjunmahishi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Propose a new feature
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- if this is a question, consider using the discussion section of this repo -->
```
Comment (st3penta) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: To ease navigation to users landing here, you could include a link to the discussion section in this comment

Comment (arjunmahishi) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: Yes. Makes sense. I will add it as soon as it's enabled. We can hold on merging this until then.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+---
+
+<!-- if this is a question, consider using the discussion section of this repo -->
+
+**Describe the bug**
+<!-- a detailed description of the bug -->
+
+**To Reproduce**
+<!-- Steps to reproduce the behavior -->
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
<!-- Steps or code snippet to reproduce the behaviour -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen -->
+
+**Screenshots/Snippets**
+<!-- If applicable, add screenshots or code snippets to help explain your problem -->
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: Please no screenshots 🙈
```suggestion
### Actual behaviour
<!-- what testify does -->


```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+about: Format to report a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- if this is a question, consider using the discussion section of this repo -->
+
+**Describe the bug**
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
## Description
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+<!-- a detailed description of the bug -->
+
+**To Reproduce**
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]:
```suggestion
## Recreate steps
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+<!-- Steps to reproduce the behavior -->
+
+**Expected behavior**
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]:
```suggestion
## Expected behaviour
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+**Description**
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]:
```suggestion
## Description
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+<!-- A clear and concise description of what feature you are proposing -->
+
+**Proposed solution**
+<!-- A clear and concise description of what you want to happen -->
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]:
```suggestion
## Proposed solution
<!-- Optionally a suggested implementation -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+**Use case**
+<!-- A clear and concise description of any alternative solutions or features you've considered -->
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/feature_request.md:19]:
```suggestion
## Use case
<!-- What is the motivation? What workarounds have you used? -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+**Any additional context**
+<!-- Add any other context or screenshots about the feature request here -->
```
Comment (brackendawson) [inline .github/ISSUE_TEMPLATE/feature_request.md:22]: I don't think we need that bit, it's kind of the same as a well written description?

**Review by brackendawson — COMMENTED** [review]
Some spelling. Some formatting. Some suggestions.

**Comment by arjunmahishi** [comment]
Thanks for the suggestion @brackendawson 
And sorry about the typos 🙈

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (brackendawson) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: `https://github.com/stretchr/testify/discussions/new/choose`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Format to report a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- if this is a question, consider using the discussion section of this repo -->
```
Comment (st3penta) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Sorry I didn't notice before, the link would be useful here too

**Comment by st3penta** [comment]
[minor] some comments don't start with capital letter: [one](https://github.com/stretchr/testify/pull/1538/files#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5R10), [two](https://github.com/stretchr/testify/pull/1538/files#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5R13), [three](https://github.com/stretchr/testify/pull/1538/files#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5R22), [four](https://github.com/stretchr/testify/pull/1538/files#diff-148870715557a13127284f8aeaa28002ed6b034268af13c5d030ab59fd078220R10)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 11 above)
Comment (arjunmahishi) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

**Comment by arjunmahishi** [comment]
> [minor] some comments don't start with capital letter: [one](https://github.com/stretchr/testify/pull/1538/files#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5R10), [two](https://github.com/stretchr/testify/pull/1538/files#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5R13), [three](https://github.com/stretchr/testify/pull/1538/files#diff-185833cb26d7ac66a4d39042fd576a820c2c2c6d05ad18973bb9c7dce77267c5R22), [four](https://github.com/stretchr/testify/pull/1538/files#diff-148870715557a13127284f8aeaa28002ed6b034268af13c5d030ab59fd078220R10)

Done

**Review by brackendawson — APPROVED** [review]
LGTM

**Pull request merged**